torch/distributed/algorithms/ddp_comm_hooks/powerSGD_hook.py

```diff
@@ -327,6 +330,10 @@ def powerSGD_hook(
         bucket (dist.GradBucket): Bucket that stores a 1D flattened gradient tensor that batches multiple per-variable tensors.
             Note that since DDP comm hook only supports single process single device mode,
             only exactly one tensor is stored in this bucket.
+        batch_tensors_with_same_shape (bool): A boolean value that when set to ``True``, compress and decompress tensors with
+            same shape in a batched operation to achieve higher parallelism. Note that you should also increase the bucket size
```

Comment:
Nit: Use "bucket size (i.e., ``bucket_cap_mb`` arg in DDP ctor)" to be more clear.